src/test/java/io/mosip/mimoto/service/IssuersServiceTest.java

```diff
@@ -86,6 +95,21 @@ public void shouldReturnIssuerDataAndConfigForTheIssuerIdIfExist() throws ApiNot
         assertEquals(expectedIssuer, issuer);
     }
 
+    @Test
+    public void shouldReturnIssuerWellknownForTheIssuerIdIfExist() throws ApiNotAccessibleException, IOException {
+        String issuerId = "test-issuer";
+        String wellKnownUrl = "https://example.com/.well-known";
+        String wellKnownResponse = "{\"issuer\":\"test-issuer\",\"authorization_endpoint\":\"https://example.com/auth\"}";
+        ResponseEntity<String> wellKnownResponseEntity=restTemplate.getForEntity(wellKnownResponse, String.class);
```

Comment:
This is not needed as restTemplate is mocked and when, then statement is handled at line no. 105